source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -26,7 +26,7 @@ export function createActionInstance(json: any, errors: Array<IValidationError>)
     if (!json["title"] && json["title"] !== "") {
         raiseParseError(
             {
-                error: Enums.ValidationError.ActionTitleCantBeNull,
+                error: Enums.ValidationError.PropertyCantBeNull,
                 message: "Action title is null"
```

Comment:
Actually, one more change if you dont mind. I think the message should probably more like "Actions should always have a title."